.ci/install_qemu.sh

```diff
@@ -91,25 +91,34 @@ if [ -f "${cidir}/${QEMU_ARCH}/lib_install_qemu_${QEMU_ARCH}.sh" ]; then
 fi
 
 main() {
-	if [ "$QEMU_ARCH" == "x86_64" ]; then
-		packaged_qemu_commit=$(get_packaged_qemu_commit)
-		short_current_qemu_commit=${CURRENT_QEMU_COMMIT:0:10}
-		if [ "$packaged_qemu_commit" == "$short_current_qemu_commit" ]; then
-			# If installing packaged qemu from OBS fails,
-			# then build and install it from sources.
-			install_packaged_qemu || build_and_install_qemu
-		else
-			build_and_install_qemu
-		fi
-	elif [ "$QEMU_ARCH" == "aarch64" ] || [ "$QEMU_ARCH" == "ppc64le" ]; then
-		packaged_qemu_version=$(get_packaged_qemu_version)
-		short_current_qemu_version=${CURRENT_QEMU_VERSION#*-}
-		if [ "$packaged_qemu_version" == "$short_current_qemu_version" ] && [ -z "${CURRENT_QEMU_COMMIT}" ]; then
-			install_packaged_qemu || build_and_install_qemu
-		else
-			build_and_install_qemu
-		fi
-	fi
+	case "$QEMU_ARCH" in
+		"x86_64")
+			packaged_qemu_commit=$(get_packaged_qemu_commit)
+			short_current_qemu_commit=${CURRENT_QEMU_COMMIT:0:10}
+			if [ "$packaged_qemu_commit" == "$short_current_qemu_commit" ]; then
+				# If installing packaged qemu from OBS fails,
+				# then build and install it from sources.
+				install_packaged_qemu || build_and_install_qemu
+			else
+				build_and_install_qemu
+			fi
+			;;
+		"aarch64"|"ppc64le")
+			packaged_qemu_version=$(get_packaged_qemu_version)
+			short_current_qemu_version=${CURRENT_QEMU_VERSION#*-}
+			if [ "$packaged_qemu_version" == "$short_current_qemu_version" ] && [ -z "${CURRENT_QEMU_COMMIT}" ]; then
+				install_packaged_qemu || build_and_install_qemu
+			else
+				build_and_install_qemu
+			fi
+			;;
+		"s390x")
+			install_packaged_qemu
```

Comment:
No, it will not build the source. As i've written in the PR-Message, building Qemu from source is currently not implemented. It will be added later, as soon as the configure_hypervisor.sh script from packaging supports s390x. The code will then look similar to x86_64.

Reply:
@grahamwhaley if this is ok for you we can add the qemu building later. Otherwise, this PR is blocked by the missing support in [configure-hypervisor.sh](https://github.com/kata-containers/packaging/blob/master/scripts/configure-hypervisor.sh). As we obtained the CI integrated, we thought to open the pr to be able at least to install the dependencies and build Kata.

Reply:
Sure, its fine - was just checking...